puzzle-utils/words/anagram.py

```diff
@@ -1,5 +1,5 @@
-from utils.dictionary import get_current_dictionary
-from utils.word import get_character_histogram, histogram_to_string
+from .utils.dictionary import get_current_dictionary
+from .utils.word import get_character_histogram, histogram_to_string
 
 class AnagramPattern:
 	num_of_letters = 26
```

Comment:
nit: Prefer either `number_of_letters` or `num_letters`. If we're shortening variable names, may as well remove prepositions